From 9c0ca0ecba834a542d9018ce319871abb38d57d2 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 22 Apr 2021 16:11:14 +0530
Subject: [PATCH] [REVERTME] allow __sF for kernelflinger build.

In Android 12 __REMOVED_IN(23) was giving error, so
temporarily disabled for bring-up. Need to be checked
in terms of API level of kernel flinger for Android12.

Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Change-Id: I4cd916006a30140b2006e93354e130b2b8683470
---
 libc/include/stdio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libc/include/stdio.h b/libc/include/stdio.h
index 9de12a47f..b53f488e1 100644
--- a/libc/include/stdio.h
+++ b/libc/include/stdio.h
@@ -69,7 +69,7 @@ extern FILE* stderr __INTRODUCED_IN(23);
 #define stderr stderr
 #else
 /* Before M the actual symbols for stdin and friends had different names. */
-extern FILE __sF[] __REMOVED_IN(23);
+extern FILE __sF[];

 #define stdin (&__sF[0])
 #define stdout (&__sF[1])
--
2.17.1

